skip_test.go

```diff
@@ -0,0 +1,51 @@
+package slogx_test
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"log/slog"
+	"testing"
+
+	"github.com/powerman/check"
+
+	"github.com/powerman/slogx"
+)
+
+func TestLogSkip(tt *testing.T) {
+	t := check.T(tt)
+	t.Parallel()
+
+	var buf bytes.Buffer
+	ctx := context.Background()
+	h := slog.NewTextHandler(&buf, &slog.HandlerOptions{AddSource: true, Level: slog.Level(8)})
+
+	slogx.LogSkip(ctx, 0, h, slog.Level(4), "message", "err", io.EOF)
+	t.Len(buf.String(), 0)
+
+	slogx.LogSkip(ctx, 0, h, slog.Level(8), "message", "err", io.EOF)
+	t.Match(buf.String(), "level=ERROR source=.*/slogx/skip_test.go:26 msg=message err=EOF")
+
+	buf.Truncate(0)
+	slogx.LogSkip(nil, 1, h, slog.Level(8), "message", "err", io.EOF)
+	t.Match(buf.String(), "level=ERROR source=.*/testing/testing.go:1595 msg=message err=EOF")
```

Comment:
Bad idea: line `1595` may change in next Go release. It will be more reliable to use extra helper function here in test.